appveyor/Push-PrefastResultsToAppveyor.ps1

```diff
@@ -0,0 +1,8 @@
+$Dir = get-childitem .\ -Recurse -Filter "*.prefast.sarif"
+
+for ($i=0; $i -lt $Dir.Count; $i++)
+{
+  $fileLocation = $Dir[$i].FullName
+  Push-AppVeyorArtifact $fileLocation
+  .\appveyor\Convert-FileUrisToGitHubUris.ps1 $fileLocation
```

Comment:
I think we should add a header (using comment string) to both ps1 files which describes their functionality.

---
In reply to: [70728786](https://github.com/Microsoft/sarif-sdk/pull/420#discussion_r70728786) [](ancestors = 70728786)